packages/core/EventManager.ts

```diff
@@ -75,7 +75,7 @@ export default class EventManager {
    * @param user
    */
   constructor(user: EventManagerUser) {
-    const appCredentials = getApps(user.credentials).flatMap((app) =>
+    const appCredentials = getApps(user.credentials, true).flatMap((app) =>
```

Comment:
The event manager is only used for installed apps so we can reduce the size of the app metadata array.